src/extensions/default/CodeFolding/Prefs.js

```diff
@@ -8,27 +8,35 @@
+        ENABLE_CODE_FOLDING         = "Enable code folding",
+        MIN_FOLD_SIZE               = "Minimum fold size",
+        MIN_FOLD_SIZE_HELP          = "Minimum number of lines to allow in a foldable range",
+        SAVE_FOLD_STATES            = "Save fold states",
+        SAVE_FOLD_STATES_HELP       = "Save fold states to disk when editor is closed and restore the folds when reopened",
+        ALWAYS_USE_INDENT_FOLD      = "Always use indent fold",
+        ALWAYS_USE_INDENT_FOLD_HELP = "Fall back to using level of indentation as a folding guideline if no range finder is found for the current mode.",
+        FADE_FOLD_BUTTONS           = "Fade fold buttons",
+        FADE_FOLD_BUTTONS_HELP      = "Hides the fold buttons unless the mouse is over the gutter",
+        MAX_FOLD_LEVEL              = "Max fold level",
+        MAX_FOLD_LEVEL_HELP         = "Used to limit the number of nested folds to find and collapse when View -> Collapse All is called or Alt is held down when collapsing. Should improve performance for large files.";
 
     //default preference values
     prefs.definePreference("enabled", "boolean", true,
-                           {name: strings.ENABLE_CODE_FOLDING, description: strings.ENABLE_CODE_FOLDING});
+                           {name: ENABLE_CODE_FOLDING, description: ENABLE_CODE_FOLDING});
     prefs.definePreference("minFoldSize", "number", 2,
-                           {name: strings.MIN_FOLD_SIZE, description: strings.MIN_FOLD_SIZE_HELP});
+                           {name: MIN_FOLD_SIZE, description: MIN_FOLD_SIZE_HELP});
     prefs.definePreference("saveFoldStates", "boolean", true,
-                           {name: strings.SAVE_FOLD_STATES, description: strings.SAVE_FOLD_STATES_HELP});
-    prefs.definePreference("alwaysUseIndentFold", "boolean", true,
-                           {name: strings.ALWAYS_USE_INDENT_FOLD, description: strings.ALWAYS_USE_INDENT_FOLD_HELP});
-    prefs.definePreference("enableRegionFolding", "boolean", true,
-                           {name: strings.ENABLE_REGION_FOLDING, description: strings.ENABLE_REGION_FOLDING});
+                           {name: SAVE_FOLD_STATES, description: SAVE_FOLD_STATES_HELP});
+    prefs.definePreference("alwaysUseIndentFold", "boolean", false,
+                           {name: ALWAYS_USE_INDENT_FOLD, description: ALWAYS_USE_INDENT_FOLD_HELP});
     prefs.definePreference("fadeFoldButtons", "boolean", false,
-                           {name: strings.FADE_FOLD_BUTTONS, description: strings.FADE_FOLD_BUTTONS_HELP});
+                           {name: FADE_FOLD_BUTTONS, description: FADE_FOLD_BUTTONS_HELP});
     prefs.definePreference("maxFoldLevel", "number", 2,
-                           {name: strings.MAX_FOLD_LEVEL, description: strings.MAX_FOLD_LEVEL_HELP});
+                           {name: MAX_FOLD_LEVEL, description: MAX_FOLD_LEVEL_HELP});
     prefs.definePreference("folds", "object", {});
```

Comment:
This should be turned into `PreferencesManager.stateManager.definePreference(...`